src/Columns/ColumnLowCardinality.h

```diff
@@ -167,6 +167,8 @@ class ColumnLowCardinality final : public COWHelper<IColumn, ColumnLowCardinalit
             callback(dictionary.getColumnUniquePtr());
     }
 
+    bool canBeInsideNullable() const override { return true; }
```

Comment:
Not quite, as I remember, it's possible to have
`LowCardinality(Nullable(...))`, not `Nullable(LowCardinality(...))`